src/display/pattern_helper.js

```diff
@@ -573,11 +573,12 @@ class TilingPattern {
     this.setFillAndStrokeStyleToContext(graphics, paintType, color);
 
     tmpCtx.translate(-dimx.scale * x0, -dimy.scale * y0);
-    graphics.transform(dimx.scale, 0, 0, dimy.scale, 0, 0);
+    graphics.transform(0, dimx.scale, 0, 0, dimy.scale, 0, 0);
```

Comment:
nit: use a constant for the 0 you added here.

Reply:
I'm adding a comment instead. We only pass `0` "manually" here, and it doesn't really matter what number we pass (it's ignored) so the fact that it's 0 doesn't have a special meaning.